### ISSUE TITLE
[Usable - Bot Framework Web Chat - Card Breakfast]: In High contrast (Aquatic and Night sky) themes, ‘Play’, ‘Mute / Unmute’, ‘more options’ buttons present on the audio are not properly visible.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#5371]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
Color sensitive users who use high contrast themes will be impacted as they get confused and consider it as an enabled element and the user's perception and experience will be impacted. 
### Note: 
User credentials should NOT be included in the bug.
### Prerequisite: 
Turn on High Contrast themes. Path: System Settings->Search and select 'Contrast themes'.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated, Suggested action

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type Card Breakfast word in type your message text box.
4. Navigate to the results.
5. Verify the issues.

### What do you expect?

In High contrast (Aquatic and Night sky) themes, ‘Play’, ‘Mute / Unmute’, ‘more options’ buttons present on the audio should be properly visible.

### What actually happened?

In High contrast (Aquatic and Night sky) themes, ‘Play’, ‘Mute / Unmute’, ‘more options’ buttons present on the audio are not properly visible.
**Observation:** This issue does not repro in Desert and Dusk modes

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/d553c235-0319-4781-a328-2a1a81491445)
![Image](https://github.com/user-attachments/assets/c4df9ab3-d359-495d-8a5f-abfbedc1b06a)
![Image](https://github.com/user-attachments/assets/2775c1e0-2b4d-4a5c-95ec-968fa18e1241)
![Image](https://github.com/user-attachments/assets/84fe46b4-e715-40be-9829-1502dbf5bc04)
![Image](https://github.com/user-attachments/assets/0c06388c-f39a-4550-83da-3f62e6a8474f)
